src/switch_core_media.c

```diff
@@ -4206,12 +4206,6 @@ SWITCH_DECLARE(switch_status_t) switch_core_media_set_codec(switch_core_session_
 				(uint32_t) a_engine->read_impl.microseconds_per_packet / 1000 != a_engine->cur_payload_map->codec_ms ||
 				rrate != a_engine->cur_payload_map->rm_rate || force > 1) {
 
-			switch_yield(a_engine->read_impl.microseconds_per_packet);
-			switch_core_session_lock_codec_write(session);
-			switch_core_session_lock_codec_read(session);
-			resetting = 1;
-			switch_yield(a_engine->read_impl.microseconds_per_packet);
```

Comment:
this `switch_yield` was a duplicate from before. but are you sure about removing `resetting = 1;`?

Reply:
Both are duplicates of line [4242](https://github.com/team-telnyx/freeswitch/pull/364/files#diff-aac8e4e42b896ccba047371b04651af489ab16ee133cdc27b38959f3b78ccea4L4242) and [4243](https://github.com/team-telnyx/freeswitch/pull/364/files#diff-aac8e4e42b896ccba047371b04651af489ab16ee133cdc27b38959f3b78ccea4L4243)